<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>704</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>950</width>
    <height>704</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>950</width>
    <height>704</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>851</width>
     <height>671</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="5" column="1">
     <widget class="QTreeWidget" name="release_notes_treeview">
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_3">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>120</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>ui/images/logo.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="new_version_label">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>A new version yyy is available.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="dl_link1_label">
        <property name="text">
         <string>zipball</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="dl_link2_label">
        <property name="text">
         <string>tarball</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Version </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="version_label">
        <property name="text">
         <string>xxx</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>24</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Impartus Downloader</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Close</set>
      </property>
      <property name="centerButtons">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>release_notes_treeview</zorder>
   <zorder>label</zorder>
   <zorder>buttonBox</zorder>
   <zorder>horizontalSpacer</zorder>
   <zorder>label_3</zorder>
   <zorder>new_version_label</zorder>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
